pandas/core/dtypes/base.py

```diff
@@ -322,3 +323,34 @@ def _is_boolean(self) -> bool:
         bool
         """
         return False
+
+    def _get_common_type(self, dtypes: List[DtypeObj]) -> Optional[DtypeObj]:
+        """
+        Return the common dtype, if one exists.
+
+        Used in `find_common_type` implementation. This is for example used
+        to determine the resulting dtype in a concat operation.
+
+        If no common dtype exists, return None. If all dtypes in the list
+        will return None, then the common dtype will be "object" dtype.
+
+        Parameters
+        ----------
+        dtypes : list of dtypes
+            The dtypes for which to determine a common dtype. This is a list
+            of np.dtype or ExtensionDtype instances.
+
+        Returns
+        -------
+        Common dtype (np.dtype or ExtensionDtype) or None
```

Comment:
what does None mean here?

Reply:
See the docstring: "If no common dtype exists, return None". 
I can certainly try to expand on what this exactly means, if you have suggestions to make it clearer

Reply:
i guess im looking for something about what are the consequences if None is returned.  I think the answer is "we try the next EADtype in `find_common_type`", but could also imagine "we end up with object dtype" or "we cast everything to numpy and get whatever `np.concatenate` gives back"

Reply:
Again, see the docstring, it is saying those things, but probably not in enough words (will try to expand that). Can you try to be more precise about what is not clear in the text above?

Reply:
yep, its pretty clear now that ive got some caffeine in me, thanks

Reply:
Expanded the docstring a bit anyway